test/integration/targets/win_scheduled_task/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: The win_scheduled_task module only works on Win2012+
+
+- name: Test in normal mode (only run on Win2012+)
+  when: "'windows-2008' not in inventory_hostname"
+  block:
+  - include: tests.yml
+
+  - name: Check the various tasks in normal mode
+    assert:
+      that:
+      - add_scheduled_task.changed == true
```

Comment:
Should run a `state: absent` first sans registered output to assert the right starting state (eg, test stopped in the middle and left the test task there).